### PR TITLE
Add mockserver use instructions

### LINKED ISSUE #270: Define a way to test the exporters using mock responses
What its need it:

- Create a directory structure that can hold the sample for GitLab, Github, Bitbucket, Gitea, Jira and Service Now
- The tool or framework need to be ligthweigth
- The tests and ramp up of the mock server need to be triggered be GitHub Actions

The initial tools proposition are:

- [Castemocks](https://github.com/castlemock/castlemock)
- [Hoverfly](https://github.com/spectolabs/hoverfly)

### PR DESCRIPTION
**Pull request by fmenesesg**
## Describe the behavior changes introduced in this PR
This PR introduces the instructions about how to start the mock server and how to create new scenarios.

## Linked Issues?

resolves #270 

## Testing Instructions

Please include any additional commands or pointers in addition to our [standard PR testing process](/docs/Development.md#testing-pull-requests).

Follow the instruction on the README file included in the mocks folder

@redhat-cop/mdt

### CODE REVIEW

File: exporters/committime/collector_github.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def get_commit_time(self, metric):
             return None
 
         url = self._prefix + metric.repo_group + "/" + metric.repo_project + self._suffix + metric.commit_hash
-        response = requests.get(url, auth=(self._username, self._token))
+        response = requests.get(url, auth=(self._username, self._token), verify=False)
```
Comment (etsauer) [inline exporters/committime/collector_github.py:31]: We'll need to parameterize this before we can merge it.

**Comment by fmenesesg** [comment]
@etsauer I add the "skip tls validation" parametrization

File: exporters/committime/collector_base.py [hunk 2]
```diff
@@ -14,14 +14,16 @@ class AbstractCommitCollector(pelorus.AbstractPelorusExporter):
     This class should be extended for the system which contains the commit information.
     """
 
-    def __init__(self, kube_client, username, token, namespaces, apps, collector_name, timedate_format, git_api=None):
+    def __init__(self, kube_client, username, token, namespaces, apps, collector_name,
+                 timedate_format, git_api=None, tls_verify=None):
```
Comment (KevinMGranger) [inline exporters/committime/collector_base.py:18]: I think we said we wanted to generalize `git_api` to something like `api_url`, right?

As much as I hate to bikeshed, changing parameter names in python is a breaking change because of keyword arguments. Not that it matters when we're the only consumer 😛 

Also, have we tested what happens with `verify=None`? The [requests](https://docs.python-requests.org/en/master/api/#requests.request) docs say it has to be a boolean or string. I think we should have it be `tls_verify=True` just to be safe.

File: mocks/README.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+export TLS_VERIFY=False
+```
+---
+**NOTE**
+
+The env TLS_VERIFY is needed because the mocks server use a self-signed certificate.
+
+---
+
+Because the commit exporters search on Openshift the information from the build, first you need to execute "oc login" against the mock server
```
Comment (KevinMGranger) [inline mocks/README.md:44]: I want to make sure I understand-- this uses mockoon's proxy mode? I think we should document that, and note that the server to proxy to is hardcoded in mockoon-- do we have some way of templating that out?

Comment (fmenesesg) [reply mocks/README.md:44]: Is not using mockoon on proxy mode, when you start the mock server that is disable, its only need to be enable when your are creating new scenarios but this is not the case

Comment (KevinMGranger) [reply mocks/README.md:44]: So then this isn't necessary for setup, right? Only for development? Then maybe we should document it elsewhere?

Comment (fmenesesg) [reply mocks/README.md:44]: To start the mock server, they are not necessary, but in this case, the instructions are to test an exporter against the mock server, in that case, you need to set up the env vars and login into the server as if you were using Openshift instance.

**Review by KevinMGranger — APPROVED** [review]
